Automattic/wordpress-activitypub
# Blocks: short-circuit early on sites that do not support blocks

Fixes #430

## Proposed changes:

This is typically only the case for sites using a custom version of WordPress, like ClassicPress.

### Other information:


## Testing instructions:

> **Note**
> You will need a site using ClassicPress to test this.

1. Install ActivityPub version 1.0.0.
2. Notice the Fatal Error in your logs.
3. Switch to this branch.
4. The Fatal Error should disappear.
